sympy/core/compatibility.py

```diff
@@ -38,12 +38,12 @@
 
 Moved modules:
     * `reduce()`
-    * `StringIO()`
-    * `cStringIO()` (same as `StingIO()` in Python 3)
+    * `cStringIO()` (uses `StringIO.cStringIO` in Python 2, same as `StingIO()`
+      in Python 3, use `io.StringIO` for normal `StringIO`)
     * Python 2 `__builtins__`, access with Python 3 name, `builtins`
 
 Iterator/list changes:
-    * `xrange` removed in Python 3, import `xrange` for Python 2/3 compatible
```

Comment:
Right, but it has been renamed in this file to `range`, so I updated the
docstring to reflect this.

On Thu, May 26, 2016, 14:57 Konstantinos Psychas <notifications@github.com>
wrote:

> In sympy/core/compatibility.py
> <https://github.com/sympy/sympy/pull/11145#discussion_r64810344>:
>
> >      * Python 2 `__builtins__`, access with Python 3 name, `builtins`
> >
> >  Iterator/list changes:
> > -    * `xrange` removed in Python 3, import `xrange` for Python 2/3 compatible
>
> What this line says is that if you are to use xrange you have to import it
> (since there is no xrange in python 3). Is it possible
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/sympy/pull/11145/files/a49a7bb24b700147112447ccfccb598cbd9c6844..561e36da0e29fbd6e6ab6d3c357b29ed418e0f21#r64810344>
>